scripts/linters/general_purpose_linter.py

```diff
@@ -213,7 +213,7 @@
     },
     {
         'regexp': re.compile(r'urllib\..*quote\('),
-        'message': 'Please use python_utils.url_quote().',
+        'message': 'Please use urllib.quote().',
         'excluded_files': ('python_utils.py', 'python_utils_test.py'),
         'excluded_dirs': ()
     },
```

Comment:
This whole thing can be removed. Since this was used to forbid users to use the library directly.

Reply:
@vojtechjelinek  Done